src/pudl/metadata/resources/ferc1.py

```diff
@@ -720,7 +722,10 @@
         "field_namespace": "ferc1",
     },
     "core_pudl__assn_ferc1_pudl_utilities": {
-        "description": "This table maps two manually assigned utility IDs: a PUDL ID and a FERC1 ID. The PUDL ID maps EIA and FERC1 utilities. The FERC1 ID maps the older DBF respondent IDs to new XBRL entity IDs. This table is generated from a table stored in the PUDL repository: src/package_data/glue/utility_id_pudl.csv",
+        "description": {
+            "additional_summary_text": "PUDL utility IDs and PUDL FERC1 utility IDs.",
```

Comment:
take this one or leave it.... i think its clear enough in the details text
```suggestion
            "additional_summary_text": "PUDL assigned utility IDs and PUDL assigned FERC1 utility IDs.",
```